### DISCUSSION TITLE
can we upgrade semaphore v2.4.1 to v2.8.x?

### DISCUSSION THREAD

**Discussion by dikayyy** [semaphoreui/semaphore#1327] [Q&A]
can we upgrade semaphore v2.4.1 to v2.8.x?

seems the database schema are not the same, how to upgrade it ?

**Comment by stefanux** [comment]
"semaphore migrate" should do the job for you?

**Comment by fiftin** [comment]
Hi @dikayyy just run new version of Semaphore. Migration will done automatically.
